lighthouse-cli/test/smokehouse/dbw-config.js

```diff
@@ -22,4 +22,8 @@ module.exports = {
       'apple-touch-icon', // pull in apple touch icon to test `LinkElements`
     ],
   },
+  audits: [
+    // Test filtering and reduce churn in test expectations at the same time! \( ﾟヮﾟ)/
+    {path: 'errors-in-console', options: {ignoredPatterns: ['Failed to load resource']}},
```

Comment:
I know switching between 8 and 10 depending on how the test is run has been really annoying, so that's good, but is there ever anything this will catch that we'll miss? e.g. 404ing on files we don't realize we're 404ing

Reply:
> is there ever anything this will catch that we'll miss? e.g. 404ing on files we don't realize we're 404ing

100% of the cases where this statement was true in the past it only mattered because this assertion failed and it needed to be updated, not because the 404 was a problem, so I guess the answer is "yes, but why should we care?"

Reply:
> 100% of the cases where this statement was true in the past it only mattered because this assertion failed and it needed to be updated, not because the 404 was a problem, so I guess the answer is "yes, but why should we care?"

I'm not sure I get it. There's been a trend to enumerating all the items in expectations since putting them behind a count or only asserting a subset has masked things or made them harder to debug (e.g. #9665). This will be ignoring them before they even get to the expectations, enumerated or not. Seems like that's worth considering for a moment?

Reply:
reverted